src/controller/records.controller.js

```diff
@@ -0,0 +1,69 @@
+import { RecordsService } from '../service/records.service.js'; // 서비스 경로
+
+export class RecordsController {
+    constructor(recordsService) {
+        this.recordsService = recordsService;
+    }
+    createRecord = async (req, res, next) => {
+        try {
+            const { groupId } = req.params;
+            const { nickname, password, exerciseType, description, time, distance, photos} = req.body;
+            const validExerciseTypes = ['RUNNING', 'CYCLE', 'SWIMMING'];
+            if (!validExerciseTypes.includes(exerciseType)) {
+                return res.status(400).json({ message: '유효한 운동 종류가 아닙니다.'})
+            }
+
+            if (!nickname || !password || !exerciseType || !time || !distance) {
```

Comment:
```suggestion
            if (!nickname || !password || !time || !distance) {
```
위에서 운동 타입에 대한 유효성 검사를 하셔서 여기서 한 번 더 확인하실 필요는 없는 것 같습니다!

Reply:
넵, 위에 유효성 검사를 새로 넣고 빼는 걸 생각 안했었네요 !